fda-model/paireddata/PairedData.cs

```diff
@@ -18,7 +18,7 @@ public class PairedData: IPairedData
         /// </summary>
         public double f(double x){
             //binary search.
-            double[] xarr = Xvals.ToArray();
+            double[] xarr = Xvals; //probably not necessary anymore.
```

Comment:
Can we kill this guy all together and just binary search Xvals?

Reply:
yep, that would be fine.